.github/workflows/deploy.yml

```diff
@@ -1,6 +1,6 @@
 name: CI
 
-on: [push, pull_request]
```

Comment:
@nitrosx 
Due to running this only on merge/push and new test names the repo config probably needs adjustment with regard to which jobs are required.

Reply:
thx, this went unnoticed, sorry!